src/pages/workspaces/workspace/Data.js

```diff
@@ -341,7 +341,8 @@ class EntitiesContent extends Component {
       deletingEntities: false,
       refreshKey: 0,
       igvFiles: undefined,
-      showIgvSelector: false
+      showIgvSelector: false,
+      igvRefGenome: ''
```

Comment:
Now that there are multiple pieces of data being set and used together, I wonder if it would be better to keep them bundled together in an object, e.g. the way they're currently being sent back from the modal. (`igvData`?)